R/plot_land.R

```diff
@@ -25,20 +25,18 @@ plotNest <- function(aLandAllocator, aScenarioInfo) {
 #' plotLandAllocation
 #'
 #' @details Plots land allocation over time
-#' @param aLandAllocator Land Allocator
 #' @param aScenarioInfo Scenario-related information, including names, logits, expectations
 #' @author KVC September 2017
-#' @importFrom readr read_csv
 #' @export
-plotLandAllocation <- function(aLandAllocator, aScenarioInfo) {
+plotLandAllocation <- function(aScenarioInfo) {
   # Silence package checks
   land.allocation <- year <- name <- NULL
 
   # Get file name
-  file <- paste0(aScenarioInfo$mOutputDir, "/output_", aScenarioInfo$mFileName, ".csv")
+  file <- paste0(aScenarioInfo$mOutputDir, "/output_", aScenarioInfo$mFileName, ".rds")
```

Comment:
It's best to put components in a path (e..g `subdir/file`) together using `file.path` instead of `paste0`.  The former automatically selects the appropriate path separator (`/` vs. `\`) for the platform it's being run on.